boards/arm/nrf52840_pca10056/doc/nrf52840_pca10056.rst

```diff
@@ -167,6 +167,54 @@ You can build and flash the examples to make sure Zephyr is running correctly on
 your board. The button and LED definitions can be found in :file:
 `boards/arm/nrf52840_pca10056/nrf52840_pca10056.dts`.
 
+Using UART1
+***********
+
+The following approach can be used when an application needs to use
+more than one UART for connecting peripheral devices:
+
+1. Add device tree overlay file to the main directory of your application
+
+.. code-block:: console
```

Comment:
For this code block to be part of the bullet list, it needs to be indented, as does the sentence after the code block.  Try this:

```
1. Add device tree overlay file to the main directory of your application:

   .. code-block:: console

      $ cat nrf52840_pca10056.overlay
      &uart1 {
        compatible = "nordic,nrf-uarte";
        current-speed = <115200>;
        status = "ok";
        tx-pin = <14>;
        rx-pin = <16>;
      };

   In the overlay file above, pin P0.16 is used for RX and P0.14 is used for TX
```